app/src/main/res/drawable/ic_add_location_black_24dp.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0">
+    <path
+        android:fillColor="#2F000000"
```

Comment:
technically, Adrianna is right. (See fillColor here: https://developer.android.com/reference/android/graphics/drawable/VectorDrawable.html). Practically, however, we typically override the fillColor with a tint, so the value of this attribute is not important. May be useful when developing; you can notice when an icon isn't the expected color.

I suggest we keep it this way, will simplify finding dev bugs.